### ISSUE TITLE
The version of safari is wrong

### ISSUE THREAD

**Issue by englebertnicolas** [wangkanai/Detection#393]
The version of safari is wrong.
When the browser is safari, the right version in the user agent is not located here "...Safari/<HERE> ..." but here ".../Version/<HERE> ...".

Example: 
User agent: "Mozilla/5.0 (iPhone; CPU iPhone OS 13_2_2 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/**13.0.3** Mobile/15E148 Safari/**604.1**"
-> Wangkanai.Detection.Browser.Version returns 604.1, but it should be 13.0.3.

(Assembly used: wangkanai.detection\2.0.0\lib\netstandard2.0\Wangkanai.Detection.dll)

**Comment by wangkanai** [comment]
Interesting, please try using the 3.0 alpha 10 and see how that works out.

**Comment by englebertnicolas** [comment]
Not able to test it because I'm currently using net core 2.1 (but as I didn't see any change about this in your code, I guess the problem will still be there).
I temporarily solved it by adding this extension:
```
    public static class WangkanaiExtensions {
        public static Version BrowserTrueVersion(this IDetection detection) {
            if (detection.Browser.Type == BrowserType.Safari) {
                var ua = detection.UserAgent.ToString();
                string version = null;
                try {
                    version = ua.Substring(ua.IndexOf("Version/") + "Version/".Length);
                    version = version.Substring(0, version.IndexOf(" "));
                } catch { }
                return version.ToVersion();

            } else {
                return detection.Browser.Version;
            }
        }
    }

```

**Comment by wangkanai** [comment]
Your code help me provide a fix to 2.0 with 2.0.1 hotfix.
Thank you very mate.
